virtual_leader_node:
  rate: 100
  set_pose_topic: set_pose    
  leader_vel_topic: leader_vel
  leader_pose_topic: leader_pose
  cmd_vel_topic: cmd_vel
  cmd_vel_timeout: 0.5